wallet/concepts/rpc-api.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 4
+description: Learn about the MetaMask JSON-RPC API.
+---
+
+# JSON-RPC API
+
+MetaMask uses the [`window.ethereum.request(args)`](../reference/provider-api.md#windowethereumrequestargs)
+method of the [provider API](provider-api.md) to wrap a JSON-RPC API.
+The JSON-RPC API contains standard Ethereum JSON-RPC API methods and MetaMask-specific methods.
+
+:::tip MetaMask API Playground
+The RPC methods are documented in the interactive
+[MetaMask JSON-RPC API Playground](/wallet/reference/eth_subscribe).
+:::
+
+Methods in the API playground may have the following tags:
+
+- **MetaMask** - These methods behave in ways specific to MetaMask, and may or may not be supported
+  by other wallets.
+  Some of these methods are documented in more detail on the [JSON-RPC reference](../reference/rpc-api.md).
+- **Restricted** - These methods are [restricted](#restricted-methods), which require requesting
+  permission using [`wallet_requestPermissions`](/wallet/reference/wallet_requestpermissions).
+- **Mobile** - These methods are only available on MetaMask Mobile.
+- **Ethereum API** - These are standard Ethereum JSON-RPC API methods.
+  See the [Ethereum wiki](https://eth.wiki/json-rpc/API#json-rpc-methods) for more information on
+  these methods.
+
+:::note
+All RPC method requests can return errors.
+Make sure to handle errors for every call to
+[`window.ethereum.request(args)`](../reference/provider-api.md#windowethereumrequestargs).
+:::
+
+## Restricted methods
+
+MetaMask introduced web3 wallet permissions in [EIP-2255](https://eips.ethereum.org/EIPS/eip-2255).
+In this permissions system, each RPC method is restricted or unrestricted.
+If a method is restricted, a dapp must request permission to call it using
+[`wallet_requestPermissions`](/wallet/reference/wallet_requestpermissions).
+Under the hood, permissions are plain, JSON-compatible objects, with fields that are mostly used
+internally by MetaMask.
+
+Outside of [Snaps restricted methods](/snaps/reference/rpc-api/#restricted-methods), the only
+restricted method is [`eth_accounts`](/wallet/reference/eth_accounts), which allows you to access
+the user's Ethereum accounts.
+More restricted methods will be added in the future.
+
+## Unrestricted methods
+
+Unrestricted methods have no corresponding permission, but they might still rely on permissions to
```

Comment:
For a second I thought this statement is saying that the unrestricted methods do not have permission (to do this or that). Maybe it reads better if you say, "Unrestricted methods do not require permission, but they might still ... "